Source/CesiumRuntime/Public/Cesium3DTileset.h

```diff
@@ -584,4 +584,5 @@ class CESIUMRUNTIME_API ACesium3DTileset : public AActor,
   bool _beforeMoviePreloadSiblings;
   int32_t _beforeMovieLoadingDescendantLimit;
   bool _beforeMovieKeepWorldOriginNearCamera;
+  std::vector<Cesium3DTiles::Tile*> _tilesToNoLongerRenderNextFrame;
```

Comment:
```
/**
 * This is used as a workaround for cesium-native#186
 *
 * The tiles that are no longer supposed to be rendered in the current 
 * frame, according to ViewUpdateResult::tilesToNoLongerRenderThisFrame,
 * are kept in this list, and hidden in the NEXT frame, because some
 * internal occlusion culling information from Unreal might prevent
 * the tiles that are supposed to be rendered instead from appearing 
 * immediately. 
 * If we find a way to clear the wrong occlusion information in the
 * Unreal Engine, then this field may be removed, and the 
 * tilesToNoLongerRenderThisFrame may be hidden immediately.
 */
 ```

Reply:
Good idea, added. I only made it a regular comment rather than a Doxygen one because Doxygen comments are only meant to document the API, right?

Reply:
It's a private member, so by default, does not appear in the Doxygen anyhow. I don't see a reason to not use `/* */`, but the difference is not sooo important right now. 

(If we wanted to document private stuff, then it would appear accordingly if it was a `/** comment */` (and it might even be that the number of asterisks is relevant here, as in `/**` vs. `/*`, but am not sure)), but that's not the case now).